src/styles/blocks/card.scss

```diff
@@ -0,0 +1,46 @@
+.card {
+  width: $card-width;
+  padding: 32px 16px 16px;
+  color: $color-black-text;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  
+  &__product-image {
+    display: block;
+    @include w-h (160, 134);
+    margin: 0 auto 40px;
+  }
+  
+  &__product-name {
+    margin-bottom: 4px;
+    @include fz-fw-lh (12, 500, 18)
```

Comment:
please give this mixin a more specific name, now I can't understand what you want to achieve at a first glance